code/src/index.js

```diff
@@ -1,6 +1,12 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import './index.css'
-import { App } from './App'
+import { App } from './components/App'
 
-ReactDOM.render(<App />, document.getElementById('root'))
+const Write = () => {
```

Comment:
Don't know what was the purpose of this component, but we shouldn't do things like that. We should pass `<App />` component directly into `ReactDOM.render()` method